lib/views/after_auth_screens/chat/chat_list_screen.dart

```diff
@@ -1,20 +1,17 @@
-// ignore_for_file: talawa_api_doc
-// ignore_for_file: talawa_good_doc_comments
-
 import 'package:flutter/material.dart';
 import 'package:talawa/views/after_auth_screens/chat/direct_chats.dart';
-import 'package:talawa/views/after_auth_screens/chat/event_chats.dart';
 import 'package:talawa/views/after_auth_screens/chat/select_contact.dart';
 
-/// ChatPage returns a stateless widget for current user Chat Page which renders
+/// ChatPage returns a stateless widget for current user Chat Page which renders.
```

Comment:
Write this doc in a better way. Don't break the line abruptly just for the sake of ending the first line.